upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,194 @@
+          - ""
+          resources:
+          - secrets
+          - pods
+          - configmaps
+          - services
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - update
+          - patch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - service-operator.aws
+          resources:
+          - "*"
+          verbs:
+          - "*"
+      deployments:
+      - name: aws-service-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: aws-service-operator
+          template:
+            metadata:
+              annotations:
+                iam.amazonaws.com/role: aws-service-operator
+              labels:
+                app: aws-service-operator
+            spec:
+              serviceAccountName: aws-service-operator
+              containers:
+              - name: aws-service-operator
+                image: awsserviceoperator/aws-service-operator:v0.0.1-alpha4
+                imagePullPolicy: Always
+                env:
+                - name: CLUSTER_NAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: CLUSTER_NAME
+                - name: REGION
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: REGION
+                - name: ACCOUNT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: ACCOUNT_ID
+                args:
+                  - server
+                  - --cluster-name=$(CLUSTER_NAME)
+                  - --region=$(REGION)
+                  - --account-id=$(ACCOUNT_ID)
+  customresourcedefinitions:
+    owned:
+    - name: cloudformationtemplates.service-operator.aws
+      version: v1alpha1
+      kind: CloudFormationTemplate
+      displayName: CloudFormationTemplate
+      description: Represents a resource for managing Cloud Formation Templates.
+    - name: dynamodbs.service-operator.aws
+      version: v1alpha1
+      kind: DynamoDB
+      displayName: DynamoDB
+      description: Represents a resource for managing a DynamoDB resource.
+    - name: ecrrepositories.service-operator.aws
+      version: v1alpha1
+      kind: ECRRepository
+      displayName: ECRRepository
+      description: Represents a resource for managing an ECR Repository.
+    - name: elasticaches.service-operator.aws
+      version: v1alpha1
+      kind: ElastiCache
+      displayName: ElastiCache
+      description: Represents a resource for managing an ElastiCache Engine.
```

Comment:
s/ElastiCache Engine/Amazon ElastiCache/

Reply:
Resolved in the latest commit.